src/components/cards/NewFeatureCard.vue

```diff
@@ -1,19 +1,36 @@
 <template>
-  <timed-card startTime="March 6, 2025" endTime="March 8, 2025">
+  <timed-card startTime="May 8, 2025" endTime="May 26, 2025">
```

Comment:
Let's keep this running until the 17th? Just so that it stays to just a week long.